server/spec/hypervisor_check_in_spec.rb

```diff
@@ -295,4 +301,295 @@ def create_virtwho_client(user)
     consumer_client = Candlepin.new(nil, nil, consumer['idCert']['cert'], consumer['idCert']['key'])
     return consumer_client
   end
+
+  # async version of method calls
+
+  def get_async_host_guest_mapping(hypervisor_id, guest_id_list)
+      guestIds = []
+      guest_id_list.each do |guest|
+          guestIds << {'guestId' => guest}
+      end
+      json = {"hypervisors" => ["name" => hypervisor_id, "hypervisorId" => {"hypervisorId" => hypervisor_id}, "guestIds" => guestIds]}
```

Comment:
One of the drivers for this was to report hypervisor hostnames. In the spec tests we're just using the hypervisor ID as the hostname, however this is exactly the same behaviour as the old API call, ideally we should have test coverage on actual hostname reporting that can't be confused with a false positive when no hypervisor hostname is specified.